modules/gke/bootstrap.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -xeuo pipefail
+
+echo "Starting GCP NVMe SSD setup"
+
+# Install required tools
+if command -v apt-get >/dev/null 2>&1; then
+  apt-get update
+  apt-get install -y lvm2
+elif command -v yum >/dev/null 2>&1; then
+  yum install -y lvm2
+else
+  echo "No package manager found. Please install required tools manually."
+  exit 1
+fi
+
+# Find NVMe devices
+SSD_DEVICE_LIST=()
+
+# Try the standard GCP pattern first
+devices=$(find /dev/disk/by-id/ -name "google-local-nvme-ssd-*" 2>/dev/null || true)
+if [ -n "$devices" ]; then
+  while read -r device; do
+    SSD_DEVICE_LIST+=("$device")
+  done <<<"$devices"
+fi
+
+# If no devices found via standard pattern, look for NVMe devices directly
+if [ ${#SSD_DEVICE_LIST[@]} -eq 0 ]; then
+  echo "No Local NVMe SSD devices found via standard pattern. Checking direct NVMe devices..."
+
+  for device in /dev/nvme*n*; do
+    # Skip if not a block device or if it's a partition
+    if [[ -b "$device" && ! "$device" =~ "p"[0-9]+ ]]; then
+      # Check if size is approximately 375GB
+      size_bytes=$(blockdev --getsize64 $device 2>/dev/null || echo 0)
+      # 375GB = approximately 402653184000 bytes
+      if ((size_bytes > 400000000000 && size_bytes < 405000000000)); then
+        echo "Found potential NVMe local SSD: $device ($((size_bytes / (1024 * 1024 * 1024))) GB)"
+        SSD_DEVICE_LIST+=("$device")
+      fi
+    fi
+  done
+fi
```

Comment:
The docs [seem pretty explicit](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd-raw#3rd-or-4th-generation_1) that the host OS should create symlinks at `/dev/disk/by-id/google-local-ssd-block[ordinal]` 

Is it necessary to check both paths, or should we just error if they can't be found at the documented path?

Reply:
Not sure but I can't get it to work that way, if I only look at the `/dev/disk/by-id/google-local-nvme-ssd-*` I end up with:

```
ERROR: No Local NVMe SSD devices found at standard path /dev/disk/by-id/google-local-nvme-ssd-*
Please verify that local SSDs were properly attached to this instance
+ echo 'ERROR: No Local NVMe SSD devices found at standard path /dev/disk/by-id/google-local-nvme-ssd-*'
+ echo 'Please verify that local SSDs were properly attached to this instance'
+ echo 'See: https://cloud.google.com/compute/docs/disks/local-ssd'
See: https://cloud.google.com/compute/docs/disks/local-ssd
+ exit 1
```

Reply:
Still not sure why, but if I do that, I end up with:

```
+ SSD_DEVICE_LIST=()
++ find /dev/disk/by-id/ -name 'google-local-ssd-*'
++ true
+ devices=
+ '[' -n '' ']'
+ echo 'ERROR: No Local SSD devices found at standard path /dev/disk/by-id/google-local-ssd-*'
+ echo 'Please verify that local SSDs were properly attached to this instance'
+ echo 'See: https://cloud.google.com/compute/docs/disks/local-ssd'
+ exit 1
ERROR: No Local SSD devices found at standard path /dev/disk/by-id/google-local-ssd-*
```

Reply:
Ah yes, the first `find` was failing as I had to add a volume mount for `/dev`, so this is not really needed.